app/routes/quotes/components/QuotePage.tsx

```diff
@@ -111,6 +125,10 @@ export default function QuotePage({
     });
   }, [history, ordering, query.approved]);
 
+  if (!loggedIn) {
+    return LoginPage;
+  }
```

Comment:
This doesn't work. We go to the 401-page because we attempt to fetch. Not sure how best to fix that...

Reply:
ah ye you're right .. I guess we should have to check before fetching 🤷🏼

Reply:
Yea, the only way to avoid running the fetch-hook at all would be to wrap the whole page in a component that checks the login. I think just checking in the hook will be easiest.